tests/unit/test_logging.py

```diff
@@ -46,3 +47,37 @@ def test_verbosity_level(
     assert (
         not unexpected_log_levels
     ), f"Unexpected log level(s): {unexpected_log_levels}"
+
+
+@pytest.mark.parametrize(
+    "verbosity_level,is_progress_bar_shown",
+    [("0", False), ("1", True)],
+)
+def test_no_progress_graphic_with_min_verbosity(
```

Comment:
Given that this test is (correctly I think) only about "No STDOUT when verbose = 0", this test could be more readable without parameterization. Then you could just assert

```
assert result.output == ''
```

Reply:
good point!